admin-console/src/main/java/org/entando/entando/apsadmin/user/UserAction.java

```diff
@@ -71,6 +71,9 @@ protected void checkAddUserUser() {
 		String username = this.getUsername();
 		String profileTypeCode = this.getProfileTypeCode();
 		try {
+			if (username.equalsIgnoreCase("guest")) {
```

Comment:
You can use `SystemConstants.GUEST_USER_NAME`

Reply:
done